sonar-batch/src/main/java/org/sonar/batch/issue/TrackedIssueAdapter.java

```diff
@@ -143,12 +152,21 @@ public String actionPlanKey() {
     return new ArrayList<>();
   }
 
+  /**
+   * @deprecated since 5.5, replaced by {@link #effort()}
+   */
   @Override
+  @Deprecated
   public Duration debt() {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)